app/templates/application.hbs

```diff
@@ -9,8 +9,6 @@
 
       <ul>
         <li>{{link-to 'Datasets' 'application'}}</li>
-        <li><a href="{{hostname}}/community-profiles">Community Profiles</a></li>
```

Comment:
The community profile link is being used. Please change to:
`<li><a href="{{hostname}}/#community-profiles">Community Profiles</a></li>`

Right now, it points to an anchor on the community selector of the home page.